docs/user_guide/hyperopt.md

```diff
@@ -27,6 +30,50 @@ hyperopt:
   metric: loss
 ```
 
+## Default Feature - Level Parameters
+
+In addition to defining hyperopt parameters for individual input or output features (like the `title` feature
+in the example above), feature level parameters can be specified for entire feature types. These parameters will
```

Comment:
1. Can you use `hyperopt parameters` instead of `feature level parameters`?
2. Can you add another example at the end of the sentence for what an entire feature type is (e.g. `text`)?